android/src/main/java/com/mparticle/react/MParticleModule.java

```diff
@@ -113,6 +113,16 @@ public void setUserAttributeArray(final String userId, final String key, final R
       }
     }
 
+    @ReactMethod
+    public void getUserAttributes(final String userId, Callback completion) {
+        MParticleUser selectedUser = MParticle.getInstance().Identity().getUser(parseMpid(userId));
+        if (selectedUser != null) {
+            completion.invoke(null, ConvertToUserIdentities(selectedUser.getUserAttributes()));
```

Comment:
did you mean to be calling ConvertToUserIdentities?

can you change this to use the async version? I believe `getUserAttributes` can take a callback